CHANGELOG.md

```diff
@@ -4,6 +4,24 @@ All notable changes to this project will be documented in this file.
 
 The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/).
 
+## Handle Granules with identical file names within a collection
+
+### Migration Notes
+
+#### CUMULUS-4146 Update granules table to include active column
+
+Please follow the instructions before upgrading Cumulus
+
+- The updates in CUMULUS-4146 require a manual update to the PostgreSQL database
+  in the production environment. Please follow the instructions in []
+
+### Added
+
+- **CUMULUS-4146**
+  - Added new column `active` to Postgres `granules` table to discern whether a duplicate granule is active or hidden
```

Comment:
Suggest 'status' or something less prescriptive than 'active' - same case as we talked about in the architecture meeting.

Reply:
Maybe?  Naming things is complicated 😓  - just suggesting that active might not be representative of what the column ends up as.